File: java/src/test/java/com/murex/LcdTest.java
package com.murex;

import org.junit.Ignore;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class LcdTest {
    @Ignore
    @Test
    public void acceptance_test() {
        int input = 120120120;
        String expected
                = "    _  _     _  _     _  _ \n"
                + "  | _|| |  | _|| |  | _|| |\n"
                + "  ||_ |_|  ||_ |_|  ||_ |_|\n";
        assertEquals(expected, Lcd.convert(input));
    }
}

File: python/tests/lcd_test.py
import unittest

from lcd.lcd import convert


class LcdTest(unittest.TestCase):

    @unittest.skip("test currently disabled")  # Comment or remove this line to enable this test case
    def test_acceptance_test(self):
        value = 120120120
        expected = [
            "    _  _     _  _     _  _ \n",
            "  | _|| |  | _|| |  | _|| |\n",
            "  ||_ |_|  ||_ |_|  ||_ |_|\n",
        ]
        self.assertEqual(''.join(expected), convert(value))


if __name__ == "__main__":
    unittest.main()
